llvm_ir_dataset_utils/analysis_tools/create_batch_files.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+STORAGE="/lustre/schandra_crpl/users/3302/ir_bc_files/"
+BATCH_PATH="/home/3302/hf_py_code/compile/codes/batch_jobs/makefile_dir/"
```

Comment:
These should be command line arguments.

Add a comment on the invocation syntax. Something like:
```
create_batche_files <storage> <batch path>
```